lib/src/klio/transforms/_helpers.py

```diff
@@ -207,6 +218,17 @@ class _KlioOutputDataMixin(object):
 
     DIRECTION_PFX = KlioIODirection.OUTPUT
 
+    def setup(self, *args, **kwargs):
+        super(_KlioOutputDataMixin, self).setup(*args, **kwargs)
+        # grab the child class name that inherits this class
+        transform_name = self.__class__.__name__
```

Comment:
wonder if we should just make `transform_name` a property on `_Klio(Input/output)DataMixin` since it's used in nearly every subclass now.